src/features/defi/providers/univ2/hooks/useUniV2LiquidityPool.ts

```diff
@@ -471,7 +471,7 @@ export const useUniV2LiquidityPool = ({
         },
       })
 
-      return feeData
+      return getFeeDataFromEstimate(feeData)
```

Comment:
Can abstract to a var so we only call this once if you feel likt it

Reply:
not sure I follow

Reply:
Disregard me, GitHub collapsing rugged me cognitively